channels/fr.m3u

```diff
@@ -149,9 +149,11 @@ http://live.lachainenormande.fr/live/lcn/livestream/playlist.m3u8
 https://lci-hls-live-ssl.tf1.fr/lci/1/hls/live_2328.m3u8
 #EXTINF:-1 tvg-id="LCP.fr" tvg-country="FR" tvg-language="French" tvg-logo="https://i.postimg.cc/rwBVxjj2/lcp.png" group-title="Legislative",LCP (480p) [Not 24/7]
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/video/xji3qy
+#EXTINF:-1 tvg-id="M6.fr" tvg-country="FR" tvg-language="French" tvg-logo="https://static.epg.best/fr/M6.fr.png" group-title="Entertainment",M6 (1080p)
+https://shls-m6-france-prod-dub.shahid.net/out/v1/c8a9f6e000cd4ebaa4d2fc7d18c15988/index.m3u8
 #EXTINF:-1 tvg-id="M6.fr" tvg-country="FR" tvg-language="French" tvg-logo="https://static.epg.best/fr/M6.fr.png" group-title="Entertainment",M6 (720p) [Not 24/7]
 http://flusonic-4.platinum-tv.com/M6/mono.m3u8?token=test
-#EXTINF:-1 tvg-id="M6International.fr" tvg-country="FR;AD;BE;LU;MC;CH" tvg-language="French" tvg-logo="https://static.epg.best/fr/M6.fr.png" group-title="Entertainment",M6 International (1080p)
+#EXTINF:-1 tvg-id="M6International.fr" tvg-country="FR;AD;BE;LU;MC;CH" tvg-language="French" tvg-logo="https://static.epg.best/fr/M6.fr.png" group-title="Entertainment",M6 International (1080p) [Offline]
```

Comment:
Says Signal Lost but it's not offline

Reply:
Signal lost appeared later. I'll keep checking and see how it goes.

Reply:
Stuck at this frame now
![image](https://user-images.githubusercontent.com/5430116/149675983-7ce12060-0d62-4632-819a-a564adf9c94d.png)

Reply:
![Capture](https://user-images.githubusercontent.com/47762050/149713652-b56676ef-c905-412e-abb9-e3f95acfafff.PNG)
working on my side

Reply:
I have "Signal Loss Detected sv1"

Reply:
> Stuck at this frame now ![image](https://user-images.githubusercontent.com/5430116/149675983-7ce12060-0d62-4632-819a-a564adf9c94d.png)

@Dum4G That's what I first saw. How do we tag the link?

Reply:
> ![Capture](https://user-images.githubusercontent.com/47762050/149713652-b56676ef-c905-412e-abb9-e3f95acfafff.PNG) working on my side

@Sphinxroot Weird, but that's M6 music not M6 International.

Reply:
I think there's no need to have a placeholder link in the playlist (M6International.fr) at least until it gets fixed and M6.fr streams regular M6 now.